cranelift/zkasm_data/mul.wat

```diff
@@ -0,0 +1,9 @@
+(module
+ (import "env" "assert_eq" (func $assert_eq (param i32) (param i32)))
+ (func $main
+	i32.const 31
+	i32.const 64516129
```

Comment:
Can you also add a test with `i64` multiplications? The 32-bit ones I believe should "just" work out of the box with plain `A * B`, but 64-bit ones AFAIU would require the `:ARITH`?

Reply:
But for this numbers ``*`` still works

Reply:
Oh, I found test which is impossible without arith: [commit](https://github.com/near/wasmtime/pull/40/commits/abb55c2be40e044d8083aabca318d79c3ebdd474). But also find a bug (=